docs/UserGuide.md

```diff
@@ -58,121 +68,131 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 </div>
 
-### Viewing help : `help`
+<!-- ### Viewing help : `help`
 
 Shows a message explaning how to access the help page.
 
 ![help message](images/helpMessage.png)
 
-Format: `help`
+Format: `help` -->
 
+### Creating a project: `project`
 
-### Adding a person: `add`
+Create a project to be tracked by the application.
 
-Adds a person to the address book.
-
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+Format: `project n/NAME d/DESCRIPTION [t/TAG]`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 A person can have any number of tags (including 0)
 </div>
 
-Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+Example: `project n/Momentum d/CS2103T Team Project t/impt`
+
+Result: Creates a project named “Momentum” with a description “CS2103T Team Project” and a tag "impt".
+
+### Updating a project: `edit`
+
+Update a project that has been previously created.
+
+Format: `edit PROJECT_ID n/NAME d/DESCRIPTION [t/TAG]`
+
+* Edits the person at the specified `PROJECT_ID`.
+* The id refers to the id number shown in the displayed project list.
+* The id **must be a positive integer** 1, 2, 3, …​
+
+Example: `project 3 n/NewMomentum d/NewDescription t\normal`
+
+Result: Updates the project with id 3. The name will be changed to “NewMomentum”, the description will be changed to “NewDescription”, the tag will be changed to normal.
+
+### Deleting a project: `delete`
+
+Deletes a project in the list.
+
+Format: `delete PROJECT_ID`
+
+* Deletes the person at the specified `PROJECT_ID`.
+* The id refers to the id number shown in the displayed project list.
+* The id **must be a positive integer** 1, 2, 3, …​
 
-### Listing all persons : `list`
+Example: `delete 2`
 
-Shows a list of all persons in the address book.
+Result: Deletes the second project in the list.
 
-Format: `list`
+### View
 
-### Editing a person : `edit`
+View different tabs of the application.
 
-Edits an existing person in the address book.
+Format: <br> `\home` <br> `\stats` <br> `\projects` <br> `\settings`
 
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
+* /home brings user to home dashboard
+* /stats brings user to Statistics tab
+* /projects brings user to Projects tab
+* /settings brings user to Settings tab
```

Comment:
I think the commands here can be highlighted once again. For example
* `/home` brings the user to home dashboard